run_complete.sh

```diff
@@ -0,0 +1,314 @@
+#!/usr/bin/env bash
+### Skript to compare the training with embedding and mc samples in it's effect on the signal strength
+
+set -e # quit on error
+shopt -s checkjobs # wait for all jobs before exiting
+export PARALLEL=1
+export USE_BATCH_SYSTEM=1
+export cluster=lxplus
+export sm_htt_analysis_dir="/portal/ekpbms3/home/mscham/sm-htt-analysis" ### local sm-htt repo !
+export cmssw_src_local="/portal/ekpbms3/home/mscham/CMSSW_10_2_14/src" ### local CMSSW !
+export batch_out_local="/portal/ekpbms3/home/mscham/batch-out"
```

Comment:
This paths should not be hardcoded here

Reply:
while it is possible to set sm_htt_analysis_dir  to `$(pwd)`, one needs to setup all the other paths by oneself anyway, so what do you propose as a solution?

Reply:
you could set it to /portal/ekpbms1/home/$(user)/... as a default, we can change it for ourselves if we want but that way it will not fail due to permissions